python/setup.py

```diff
@@ -182,9 +182,10 @@ def get_option_from_sys_argv(option_str):
     extra_compile_args = []
 
     if sys.platform != 'win32':
-        extra_compile_args.append('-Wno-write-strings')
-        extra_compile_args.append('-Wno-invalid-offsetof')
-        extra_compile_args.append('-Wno-sign-compare')
+      extra_compile_args.append('-Wno-write-strings')
+      extra_compile_args.append('-Wno-invalid-offsetof')
```

Comment:
You changed the indentation of the whole block here. Please restore the original indentation (4 spaces per level)

Reply:
Done.